test/looping.js

```diff
@@ -128,14 +128,14 @@ describe('Looping', function() {
   });
 
   it('should ignore loop with invalid through attribute (looping/undefinedObjectLoop.html)', function(done) {
+    teddy.setVerbosity(2);
     assert.equalIgnoreSpaces(teddy.render('looping/undefinedObjectLoop.html', model), '<div></div>');
+    teddy.setVerbosity(1);
```

Comment:
This is related to bringing back the invalid through var warning in #120.

I'm setting verbosity to 2 so the warning prints and that code line is covered